src/main/resources/default/templates/biz/jobs/jobs.html.pasta

```diff
@@ -13,10 +13,12 @@
             <i:block name="actions">
                 <i:if test="isFrameworkEnabled('biz.scheduler-jdbc') || isFrameworkEnabled('biz.scheduler-mongo')">
                     <t:permission permission="permission-manage-scheduler">
-                        <a class="btn btn-link pr-2" href="/jobs/scheduler">
-                            <i class="far fa-clock"></i>
-                            <span class="pl-1 d-none d-md-inline-block">@i18n("SchedulerEntry.plural")</span>
-                        </a>
+                        <div>
+                            <a class="btn btn-link" href="/jobs/scheduler">
+                                <i class="far fa-clock"></i>
+                                <span class="pl-1 d-none d-md-inline-block">@i18n("SchedulerEntry.plural")</span>
+                            </a>
+                        </div>
```

Comment:
is the pr2 removed intentionally?

Reply:
yes, otherwise the icon isn´t centered. With the div the button and the gap is big enough.